skimage/feature/texture.py

```diff
@@ -291,3 +293,130 @@ def local_binary_pattern(image, P, R, method='default'):
     image = np.ascontiguousarray(image, dtype=np.double)
     output = _local_binary_pattern(image, P, R, methods[method.lower()])
     return output
+
+def multiblock_local_binary_pattern(int_image, x, y, width, height):
```

Comment:
@warmspringwinds My only concern here is re-work. I can understand that you need to run the evaluation phase to be sure of this implementation, but I think the MB-LBP is a pretty straight forward technique and not much can go wrong with it. 

@stefanv Thoughts ?